### DISCUSSION TITLE
Does someone know the package/app name of the dir selector used for SAF?

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#10481] [Q&A] [answered]
One support mail, there are some users where this system tool (which is shown during folder selection for SAF) has some kind of bug/problem. E.g. its showing nothing at all, or its fixed on "recent files" without any interaction possibility.

I would like to ask the user to try to reset that tool to default / empty its cache or similar action. Does it have a specific name in the android system under which the user is able to find it in his installed app or system app list?

Example:
![grafik](https://user-images.githubusercontent.com/949669/116090628-5ec8d700-a6a4-11eb-925d-a08f9d97f172.png)

**Comment by eddiemuc** [comment, marked as answer]
No idea. I always assumed this is an android system base thing

**Reply by moving-bits** [reply]
Same for me

**Reply by eddiemuc** [reply]
One thing came to my.mind: on some devices (emulators for older android versions) I found that the displayed selector only showed the empty "recent" panel to me first (a bit like in the screenshot). I had to enable "show internal storage" in a context menu (three dots in upper right corner) first before I could select anything. 

Maybe something similar here?
